### DISCUSSION TITLE
USB 3.0 For monitor

### DISCUSSION THREAD

**Discussion by ghost** [devboloji/Infinix-Hackintosh-Opencore-Guide#20] [Q&A]
Did anyone found usb 3.0 dock that works for this efi ?

I got a trouble finding usb 3.0 that works with extend mode for my second monitor

**Comment by devboloji** [comment]
@CreativeKittens buy a usb 3.0 to hdmi converter which supports MacOS.

**Reply by ghost** [reply]
Yeah, but it's hard to find that support mac os, any brand suggestion?

**Reply by devboloji** [reply]
@CreativeKittens check all comments and discussions in this repository. One guy bought converter and it's working fine.

**Comment by devboloji** [comment]
@CreativeKittens Nah, Just go with your budget and try it.,😉😉
